cli/src/commands/mod.rs

```diff
@@ -91,6 +91,8 @@ enum Command {
     Diffedit(diffedit::DiffeditArgs),
     Duplicate(duplicate::DuplicateArgs),
     Edit(edit::EditArgs),
+    #[command(alias = "obslog")]
+    Evolog(evolog::EvologArgs),
```

Comment:
Should there be an alias for `evolution-log` as well?

I'm OK with or without it, but if we don't have it, the commit description and the CHANGELOG need editing.

Reply:
Oops, thanks for catching that.